fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -1025,6 +1025,19 @@ public ParseNode visitCreateMaterializedViewStatement(
             }
             return new CreateMaterializedViewStmt(tableName.getTbl(), queryStatement, properties);
         }
+        if (refreshSchemeDesc instanceof AsyncRefreshSchemeDesc) {
+            AsyncRefreshSchemeDesc asyncRefreshSchemeDesc = (AsyncRefreshSchemeDesc) refreshSchemeDesc;
+            if (asyncRefreshSchemeDesc.getIntervalLiteral() != null &&
+                    asyncRefreshSchemeDesc.getIntervalLiteral().getUnitIdentifier() != null) {
+                String description = asyncRefreshSchemeDesc.getIntervalLiteral().getUnitIdentifier().getDescription();
+                if (!("SECOND".equalsIgnoreCase(description) || "MINUTE".equalsIgnoreCase(description) ||
```

Comment:
extract this logic into a function and call it in both create and alter.
And more, push supported units into a unmodifiable set and just use find to judge whether the unit is supported.
And more, add an UT for this change.